Registry/RegistryParse.py

```diff
@@ -288,19 +346,19 @@ def hive_name(self):
     def first_hbin_offset(self):
         """
         Get the buffer offset of the first HBINBlock as an unsigned integer.
-        Note: always returns 0x1000 nothing else is known.
+        Note: always returns 0x1000, nothing else is possible.
         """
         return 0x1000
 
-    def last_hbin_offset(self):
```

Comment:
I don't think so, because the `last_hbin_offset()` method doesn't return the offset of the last hive bin, the returned value points to the last hive bin only when this hive bin is 4096 bytes in length (equal to the length of the base block). When the last hive bin is larger than 4096 bytes, the `last_hbin_offset()` method points inside the last hive bin. So any code using the old method is buggy.